website/source/docs/guides/production.html.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: "docs"
+page_title: "Production Hardening"
+sidebar_current: "docs-guides-prod"
+description: |-
+  Production hardening of a Vault setup.
+---
+
+# Production Hardening
+
+This guide provides guidance on best practices for a production hardened deployment of Vault.
+The recommendations are based on the [security model](/docs/internals/security.html) and focus on defense in depth.
+
+* **End-to-End TLS**. Vault should always be used with TLS in production. If intermediate load balancers or reverse proxies are used to front Vault, they should _not_ terminate TLS. This way traffic is always encrypted in transit to Vault and minimizes risks introduced by intermediate layers.
```

Comment:
It might be worth noting an exception to localhost communication. It's kinda silly to encrypt local connections.